build.gradle

```
@@ -8,7 +8,7 @@ buildscript {
     jcenter()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.2.0'
+    classpath 'com.android.tools.build:gradle:4.1.0'
```

Looks like in November 4.1.1 came out, probably after you'd initially started working on this.

```suggestion
    classpath 'com.android.tools.build:gradle:4.1.1'
```